docs/docs/k8s/ingress.md

```diff
@@ -350,6 +352,41 @@ spec:
     secretName: example-tls
 ```
 
+## TCP Endpoints
+
+The example route below defines a route providing a [tunneled TCP connection](/docs/tcp/readme.md) to an upstream service listening for non-web traffic:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: tcp-example
+  annotations:
+    ingress.pomerium.io/tcp_upstream: "true"
+spec:
+  ingressClassName: pomerium
+  rules:
+    - host: "tcp.localhost.pomerium.io"
+      http:
+        paths:
+          - pathType: ImplementationSpecific
+            backend:
+              service:
+                name: tcp-service
+                port:
+                  name: app
+```
+
+The important points to note in this example:
+
+- The annotation `ingress.pomerium.io/tcp_upstream:` is set to `"true"`,
+- `spec.rules.[].http.paths.[].path` is omitted,
+- `spec.rules.[].http.paths.[].pathType` is set to `ImplementationSpecific`,
```

Comment:
do we need change `is` into `should` ? 
cause all three conditions should be met for the route to be considered a TCP route.

Reply:
I see where you're coming from, but since we're referring to the specific example above, these things is/are.